src/main/java/com/commercetools/payment/handler/CommercetoolsExecutePaymentsController.java

```diff
@@ -26,13 +30,15 @@ public CommercetoolsExecutePaymentsController(@Nonnull StringTrimmerEditor strin
 
     @RequestMapping(
             method = RequestMethod.POST,
+            consumes = APPLICATION_JSON_VALUE,
+            produces = APPLICATION_JSON_VALUE,
             value = "/{tenantName}/commercetools/execute/payments/")
     public ResponseEntity executePayments(@PathVariable String tenantName,
-                                                 @RequestParam String paypalPlusPaymentId,
-                                                 @RequestParam String paypalPlusPayerId) {
+                                          @Valid @RequestBody PaypalPlusExecuteParams params,
+                                          BindingResult result) {
         PaymentHandleResponse paymentHandleResponse = paymentHandlerProvider
                 .getPaymentHandler(tenantName)
-                .map(paymentHandler -> paymentHandler.executePayment(paypalPlusPaymentId, paypalPlusPayerId))
+                .map(paymentHandler -> paymentHandler.executePayment(params.getPaypalPlusPaymentId(), params.getPaypalPlusPayerId()))
```

Comment:
If he does not provide both params, he will get back 400 and the following message. It's not the best but it provides all the info:

```json
{
    "errorMessage": "Field error in object 'paypalPlusExecuteParams' on field 'paypalPlusPayerId': rejected value [null]; codes [NotBlank.paypalPlusExecuteParams.paypalPlusPayerId,NotBlank.paypalPlusPayerId,NotBlank.java.lang.String,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [paypalPlusExecuteParams.paypalPlusPayerId,paypalPlusPayerId]; arguments []; default message [paypalPlusPayerId]]; default message [may not be empty]. Field error in object 'paypalPlusExecuteParams' on field 'paypalPlusPaymentId': rejected value [null]; codes [NotBlank.paypalPlusExecuteParams.paypalPlusPaymentId,NotBlank.paypalPlusPaymentId,NotBlank.java.lang.String,NotBlank]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [paypalPlusExecuteParams.paypalPlusPaymentId,paypalPlusPaymentId]; arguments []; default message [paypalPlusPaymentId]]; default message [may not be empty]"
}
```